### PR TITLE
Enterprise first

### PR DESCRIPTION
**Pull request by m-kharbat**
- [x] license.enterprise true by default

- [x] license.ident runtime setting

- [x] add sys check for unlicensed enterprise edition

- [x] add warning for unlicensed enterprise 

- [ ] add redirect for cluster check link

### CODE REVIEW

File: sql/src/test/java/io/crate/integrationtests/TableCompatibilitySysChecksTest.java [hunk 1]
```diff
@@ -55,6 +55,9 @@ public void shutdown() throws IOException {
     @Test
     public void testUpgradeRequired() throws Exception {
         startUpNodeWithDataDir("/indices/data_home/cratedata_upgrade_required.zip");
+        //add license ident setting to avoid unlicensed enterprise check
+        execute("set global transient 'license.ident' = 'my-awesome-key'");
```
Comment (chaudum) [inline sql/src/test/java/io/crate/integrationtests/TableCompatibilitySysChecksTest.java]: settings are identifiers, not strings (or did this change?)

```sql
set global transient "license.ident" = 'my-awesome-key'
```

File: blackbox/docs/configuration.txt [hunk 2]
```diff
@@ -1852,10 +1852,17 @@ Enterprise License
...
 
 **license.enterprise**
-  | *Default:*  ``false``
+  | *Default:*  ``true``
   | *Runtime:*  ``no``
 
-  Setting this to ``true`` enables the `Enterprise Edition`_ of CrateDB.
+  Setting this to ``false`` disabled the `Enterprise Edition`_ of CrateDB.
+
+**license.ident**
```
Comment (chaudum) [inline blackbox/docs/configuration.txt]: didn't we say that we don't document this setting?

File: sql/src/test/java/io/crate/integrationtests/TableCompatibilitySysChecksTest.java (hunk 1 above)
Comment (m-kharbat) [inline sql/src/test/java/io/crate/integrationtests/TableCompatibilitySysChecksTest.java]: @chaudum both statements work http://g.recordit.co/a2apWMnVvP.gif

```set global transient "license.ident" = 'my-awesome-key'
set global transient 'license.ident' = 'my-awesome-key'```

File: blackbox/docs/configuration.txt (hunk 2 above)
Comment (m-kharbat) [inline blackbox/docs/configuration.txt]: i will remove it

File: CHANGES.txt [hunk 3]
```diff
@@ -15,6 +15,8 @@ the appropriate section of the docs.
 Breaking Changes
 ================
 
+ - The ``license.enterprise`` setting is set to ``true`` by default.
```
Comment (bogensberger) [inline CHANGES.txt]: i would also mention what's breaking here, e.g: "If you start crate enterprise without a valid enterprise license you are entering the unlicensed mode"

Comment (bogensberger) [inline CHANGES.txt]: i would also mention what's breaking here, e.g: "If you start crate enterprise without a valid enterprise license you are entering the unlicensed mode"

**Pull request closed without merging**